overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -135,6 +135,8 @@ func (s *servicectlSuite) TearDownTest(c *C) {
 	s.BaseTest.TearDownTest(c)
 }
 
+// FIXME: uncomment once the feature is fixed to work on install/refresh
+/*
```

Comment:
another approach would be do use c.Skip("reason") from SetupTest

Reply:
Thanks for suggestion, that's much cleaner. Done.